frontend/src/components/app/app.tsx

```diff
@@ -69,15 +69,21 @@ const App: FC = () => {
           path={AppRoute.UAM_CONFIGURE_GROUP}
           element={
             <AuthorizedWrapper>
-              <UAMConfigureGroup />
```

Comment:
Create a AuthorizedProtectedRoute
That includes AuthorizedWrapper + UAMConfigureGroup